src/Stratis.Bitcoin/Consensus/PerformanceCounters/ConsensusManager/ConsensusManagerPerformanceSnapshot.cs

```diff
@@ -34,18 +43,24 @@ public override string ToString()
 
     public class ExecutionsCountAndDelay
     {
-        public int TotalExecutionsCount;
-        public long TotalDelayTicks;
+        private int totalExecutionsCount;
+        private long totalDelayTicks;
 
         public ExecutionsCountAndDelay()
         {
-            this.TotalExecutionsCount = 0;
-            this.TotalDelayTicks = 0;
+            this.totalExecutionsCount = 0;
+            this.totalDelayTicks = 0;
         }
 
         public double GetAvgExecutionTimeMs()
         {
-            return Math.Round((this.TotalDelayTicks / (double)this.TotalExecutionsCount) / 1000.0, 4);
+            return Math.Round((this.totalDelayTicks / (double)this.totalExecutionsCount) / 10000.0, 4);
```

Comment:
This is a nice find, I see that `StopwatchDisposable` utilises `TimeSpan.Ticks` on disposal, so 10 000 would indeed appear to be the correct conversion factor to give milliseconds.